core-bundle/contao/templates/backend/be_main.html5

```diff
@@ -25,6 +25,16 @@ $this->containerAttributes = $this
     ->mergeWith($this->containerAttributes)
 ;
 
+$this->mainAttributes = $this
+    ->attrs($this->attributes)
```

Comment:
```suggestion
    ->attrs()
```

What is `$this->attributes`? It's not present in the Twig version of this template.